pocs/tests/test_observatory.py

```diff
@@ -1,17 +1,19 @@
 import os
-import pytest
-
 import time
+
+import pytest
 from astropy import units as u
 from astropy.time import Time
 
-from pocs import hardware
 import pocs.version
+from pocs import hardware
+from pocs.camera import create_cameras_from_config
 from pocs.observatory import Observatory
+from pocs.scheduler import create_scheduler_from_config
 from pocs.scheduler.dispatch import Scheduler
 from pocs.scheduler.observation import Observation
-from pocs.camera import create_cameras_from_config
 from pocs.utils import error
+from pocs.utils.location import setup_site_location_details_from_config
```

Comment:
```suggestion
from pocs.utils.location import create_location_from_config
```

Let's just call this `create_location_from_config` even though it returns more than just a strict `location`.